tests/spread/general/big/rockcraft.yaml

```diff
@@ -35,19 +35,6 @@ parts:
       - path: dir2/3.txt
         mode: "755"
 
-  usrmerge-part:
```

Comment:
I moved this usrmerge check to its own spread test because I think it's "important" enough. Also it messes with common places like `${CRAFT_PART_INSTALL}/bin` so I feared these parts might start interfering with each other